Library/Homebrew/formula_auditor.rb

```diff
@@ -218,17 +210,6 @@ def audit_license
       end
     end
 
-    # TODO: try to remove these, it's not a good user experience
```

Comment:
Would be good to port over this `TODO` somehow?

Reply:
Was thinking about opening an issue in homebrew/core. Thoughts? We can't really have comments in JSON files and it probably doesn't belong in this repo if the exception list is gone

Reply:
Could maybe just be another CODEOWNERS for now?

Reply:
Maybe I'm misunderstanding what you mean because I don't really see how having this in `CODEOWNERS` would help. It would notify whoever is an "owner" _when_ the list is changed but won't _prompt_ anyone to try to remove instances.  It could, though, serve as an extra check to make sure new formulae aren't added. So maybe both? If so, should the owners be you and TSC as well?

Side note, `@Homebrew/tsc` includes you (and will always include the project leader), right? So no need to add both for `CODEOWNERS` changes (unless you'd rather be explicit).